server/realTime/scripts/Network.js

```diff
@@ -202,29 +220,68 @@ Protocol.prototype.getRegionSearchFromSat = function getRegionSearchFromSat (sat
 	}
 };
 
-Protocol.prototype.isInsideProtectedRegion = function isInsideProtectedRegion (owner, satObjects, room) {
+Protocol.prototype.getUsersProtectedRegions = function getUsersProtectedRegions (user, room) {
```

Comment:
This name is rather unclear. Should probably be renamed to something like getProtectedRegionsOwnedBy